app/assets/javascripts/view-claim.js

```diff
@@ -1,111 +1,151 @@
-$(function () {
-  totalApproved()
+function totalApproved () {
+  let approvedCost = 0
+  const manuallyProcessed = 0
+  let deduction = 0
 
-  if (document.getElementById('additional-info-reject') !== null) {
-    if (document.getElementById('additional-info-reject').value === 'Other') {
-      document.getElementById('additional-info-reject-manual').style.display = 'block'
-      document.getElementById('manual-label').style.display = 'block'
-    }
+  function isValidValue (value) {
```

Comment:
Sorry I seem to spot things after re-reading. Is `value` a string or Number? `isNan` doesn't make sense if you're also using `trim` on a string? `isNaN` will be OK with `1e3` for example.